internal/apiserver/route_get_metrics.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/hyperledger/firefly/internal/config"
+	"github.com/hyperledger/firefly/internal/i18n"
+	"github.com/hyperledger/firefly/internal/oapispec"
+	"github.com/hyperledger/firefly/pkg/database"
+	"github.com/hyperledger/firefly/pkg/fftypes"
+)
+
+var getMetrics = &oapispec.Route{
+	Name:   "getMetrics",
+	Path:   "namespaces/{ns}/metrics",
```

Comment:
I think `metrics` is actually a confusing name for this endpoint, especially considering the recently added Prometheus metrics. This is really just performing a count and splitting it into buckets. Based on a quick poll of the desks around me, possibly `histogram` is a good name for it instead?

Reply:
Not just yet, but let's keep that in mind for the near future.